pcp/PCPDynamicColumn.c

```diff
@@ -166,7 +167,7 @@ static void PCPDynamicColumn_parseFile(PCPDynamicColumns* columns, const char* p
       } else if (value && column && String_eq(key, "description")) {
          free_and_xStrdup(&column->super.description, value);
       } else if (value && column && String_eq(key, "width")) {
-         column->super.width = strtoul(value, NULL, 10);
+         column->super.width = atoi(value);
```

Comment:
I can't understand why the change from `strtoul` to `atoi` here, especially that `atoi` will make the behavior undefined for values that are out of representable range.

Reply:
FWIW, due to lack of checking the return value/error code, the former was somewhat UB too … ;-)